src/System.Private.CoreLib/shared/System/Globalization/TextElementEnumerator.cs

```diff
@@ -51,7 +51,7 @@ public bool MoveNext()
             return true;
         }
 
-        public object Current => GetTextElement();
+        public object? Current => GetTextElement();
```

Comment:
Why can this return nullable?

Reply:
Annotating IEnumerator.  I added a TODO-NULLABLE, as we should be able to constrain this to be non-nullable.